### PR TITLE
add python-zeroconf

### PR DESCRIPTION
**Pull request by totaam**
Notes:
* this is based on the archlinux PKGBUILD
* the python2 variant cannot be updated past the 0.19.x series

### CODE REVIEW

File: mingw-w64-python3-zeroconf/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Maintainer: Antoine Martin <antoine@xpra.org>
+# Contributor: Jelle van der Waa <jelle@archlinux.org>
+# Contributor: Maxim Andersson <thesilentboatman@gmail.com>
+
+pkgname=python-zeroconf
```
Comment (Alexpux) [inline mingw-w64-python3-zeroconf/PKGBUILD]: This is wrong package name for mingw package. Look other packages

File: mingw-w64-python2-zeroconf/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Maintainer: Antoine Martin <antoine@xpra.org>
+# Contributor: ZaZam <zazaamm æt gmail døt com>
+# Contributor: Yegorius <yegorius@domic.us>
+
+pkgname=python2-zeroconf
```
Comment (Alexpux) [inline mingw-w64-python2-zeroconf/PKGBUILD]: This is wrong package name for mingw package. Look other packages

File: mingw-w64-python2-zeroconf/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pkgdesc="A pure python implementation of multicast DNS service discovery"
+arch=('any')
+url="https://github.com/jstasiak/python-zeroconf"
+license=('LGPL')
+depends=("${MINGW_PACKAGE_PREFIX}-python2"
+         "${MINGW_PACKAGE_PREFIX}-python2-netifaces"
+	 "${MINGW_PACKAGE_PREFIX}-python2-six"
+	 "${MINGW_PACKAGE_PREFIX}-python2-enum34")
+makedepends=("${MINGW_PACKAGE_PREFIX}-python2-setuptools")
+source=("https://github.com/jstasiak/${_srcname}/archive/${pkgver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-python2-zeroconf/PKGBUILD]: You need download sources into named archive

File: mingw-w64-python3-zeroconf/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+arch=('any')
+url="https://github.com/jstasiak/python-zeroconf"
+license=('LGPL')
+depends=("${MINGW_PACKAGE_PREFIX}-python3"
+         "${MINGW_PACKAGE_PREFIX}-python3-netifaces"
+         "${MINGW_PACKAGE_PREFIX}-python3-six"
+         "${MINGW_PACKAGE_PREFIX}-python3-ifaddr")
+makedepends=("${MINGW_PACKAGE_PREFIX}-python3-setuptools")
+checkdepends=("${MINGW_PACKAGE_PREFIX}-python3-pytest")
+source=("https://github.com/jstasiak/${pkgname}/archive/${pkgver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-python3-zeroconf/PKGBUILD]: You need download sources into named archive

File: mingw-w64-python3-zeroconf/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+build() {
+  cd "${pkgbase}-${pkgver}"
+  python3 setup.py build
+}
+
+package() {
+  cd "${pkgbase}-${pkgver}"
+
+  python3 setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1
```
Comment (Alexpux) [inline mingw-w64-python3-zeroconf/PKGBUILD]: Mingw dont use /usr prefix. Look into other mingw python packages

File: mingw-w64-python2-zeroconf/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+prepare() {
+  cd "${srcdir}/${_srcname}-${pkgver}"
+
+  # No need for enum-compat if depend on enum34
+  sed -i "s/'enum-compat',//" setup.py
+}
+
+package() {
+  cd "${srcdir}/${_srcname}-${pkgver}"
+  python2 setup.py install --prefix=/usr --root="$pkgdir" --optimize=1
```
Comment (Alexpux) [inline mingw-w64-python2-zeroconf/PKGBUILD]: Mingw dont use /usr prefix. Look into other mingw python packages

File: mingw-w64-python3-zeroconf/PKGBUILD (hunk 1 above)
Comment (totaam) [reply mingw-w64-python3-zeroconf/PKGBUILD]: fixed in updated pull request

**Pull request merged**